lib/blockscore/question_set.rb

```diff
@@ -16,7 +16,7 @@ def score(answers = nil)
     private
 
     def rescore(answers)
-      @attributes = post("#{endpoint}/#{id}/score", answers: answers).attributes
+      capture_attributes(post("#{endpoint}/#{id}/score", answers: answers))
```

Comment:
I hadn't thought of using `Pathname` because it wasn't a `file` path, but with endpoint being a URI that seems like a suitable alternative.